terraform/nomad-whisper/user-data/postinstall-runner-nomad-whisper-oracle.sh

```diff
@@ -0,0 +1,17 @@
+# nomad customizations
+
+# with no overrides, provision based on the variables below using default_provision
+# default_provision takes the following variables ANSIBLE_PLAYBOOK, ANSIBLE_VARS and HOST_ROLE (defaults to $SHARD_ROLE)
+export ANSIBLE_PLAYBOOK="nomad-client.yml"
+
+. /usr/local/bin/oracle_cache.sh
+[ -z "$CACHE_PATH" ] && CACHE_PATH=$(ls /tmp/oracle_cache-*)
+export POOL_TYPE_TAG="pool_type"
+export POOL_TYPE=$(cat $CACHE_PATH | jq -r --arg POOL_TYPE_TAG "$POOL_TYPE_TAG" ".[\"$POOL_TYPE_TAG\"]")
```

Comment:
I think we can just hardcode the pool type to whisper here, since we won't ever run anything else with this postinstall runner.